python/configuration.md

```diff
@@ -1,250 +1,13 @@
-interacting with AWS.
-
-These environment variables are recommended for use if your system credentials
-are fudged for the sake of your tests.
-
-### `YOURBASE_DEBUG`
-
-- **Type:** bool-ish (`0`, `false`, `off`, `1`, `true`, `on`)
-- **Default:** off
-
-When on, YourBase will report significantly more internal information to stdout,
-stderr, and the file returned by this expression:
-
-```sh
-echo ${XDG_STATE_HOME-~/.local/state}/yourbase/python.log
-```
-
-### `YOURBASE_DISABLE`
-
-- **Type:** bool-ish (`0`, `false`, `off`, `1`, `true`, `on`)
-- **Default:** off
-
-When on, YourBase will not load.
-
-Enabling this setting then manually attaching to a test framework using
-`yourbase.attach` produces undefined behavior.
-
-### `YOURBASE_IGNORE_LOCAL_CACHE`
-
-- **Type:** bool-ish (`0`, `false`, `off`, `1`, `true`, `on`)
-- **Default:** off
-
-When on, YourBase will not read or write dependency graphs on the filesystem. If
-[`YOURBASE_REMOTE_CACHE`](#yourbase_remote_cache) is set, it will still be used
-as normal.
-
-This setting can be used if the local cache is expected to be poisoned. This can
-happen if cohorting is used against a local cache.
-
-### `YOURBASE_SYNC_DIRTY`
-
-- **Type:** bool-ish (`0`, `false`, `off`, `1`, `true`, `on`)
-- **Default:** off
-
-When on, YourBase will [synchronize graphs](#yourbase_remote_cache) even if the
-Git working tree is dirty.
-
-This setting is not recommended for use if you plan to use YourBase on developer
-machines, as it will poison the remote cache.
-
-If you only plan to use YourBase in CI, this setting can help you overcome
-situations where you need your working tree to be dirty while building, and you
-know the dirtiness will not affect the dependency graph.
-
-If that situation does not apply to you, do not enable this setting.
-
-### `YOURBASE_TELEMETRY`
-
-- **Type:** bool-ish (`0`, `false`, `off`, `1`, `true`, `on`)
-- **Default:** on
-
-When on, YourBase will send anonymized telemetry data to `api.yourbase.io` over
-HTTPS for the purposes of improving the product.
-
-Telemetry data **never** includes your code.
-
-### `YOURBASE_TIMID`
-
-- **Type:** bool-ish (`0`, `false`, `off`, `1`, `true`, `on`)
-- **Default:** off
-
-When on, YourBase will use a slower tracing algorithm that is less prone to
-conflicts with other packages than the default. Only enable this if you
-experience issues with the default algorithm.
-
-### `YOURBASE_WORKDIR`
-
-- **Type:** absolute or relative path
-- **Default:** `.`
-
-The directory YourBase should treat as the project directory. Only code in this
-directory, or one of its descendants, is guaranteed to be traced. You usually do
-not need to change this.
-
-[hi]: mailto:hi@yourbase.io
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Redirecting to https://docs.yourbase.io/environment-variables</title>
+<meta http-equiv="refresh" content="0; URL=https://docs.yourbase.io/environment-variables">
```

Comment:
For example:
```
<meta http-equiv="refresh" content="0;url=environment-variables.html">
<p>This page has <a href="environment-variables.html">moved</a>.</p>
```

Reply:
Done. Please check

Reply:
Also,
> Ah, Ross pointed out that for the files we are moving under cli/, we should ensure there's a redirect in place (for SEO purposes).
Since adding redirects for files under cli/ would keep the clutter that I was trying to resolve, I've moved those files back to where they were - so they should appear untouched in the PR.